Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs

```diff
@@ -121,9 +140,18 @@ private WinType RuleWinType
     private EntityUid? _nukieShuttle;
     private EntityUid? _targetStation;
 
+    private bool _declarationMade = false;
+    private bool _leftOutpost = false;
+
+    public TimeSpan _gameruleStartTime;
+    public TimeSpan _declarationTime;
+    public ref readonly TimeSpan GameruleStartTime => ref _gameruleStartTime;
+    public ref readonly TimeSpan DeclarationTime => ref _declarationTime;
+
     public override string Prototype => "Nukeops";
 
     private NukeopsRuleConfiguration _nukeopsRuleConfig = new();
+    public ref readonly NukeopsRuleConfiguration Config => ref _nukeopsRuleConfig;
```

Comment:
wut

Reply:
Well, I need a way to get config values in other systems

Reply:
That's doable without ref readonly fields?

Reply:
I can do something like this:
```csharp
public NukeopsRuleConfiguration Config
    {
        get { return _nukeopsRuleConfig; }
    }
```